MapboxMobileEvents/MMEConstants.h

```diff
@@ -135,13 +135,57 @@ extern NSString * const MMEEventUnknown;
 
 extern NSString * const MMEResponseKey;
 
-// SDK event source
+/*! @brief SDK event source */
 extern NSString * const MMEEventSource;
 
-// Log reporter HTML
-extern NSString * const MMELoggerHTML;
-extern NSString * const MMELoggerShareableHTML;
+#pragma mark - mobile.crash Keys
 
-@interface MMEConstants: NSObject
+extern NSString * const MMEEventMobileCrash;
+extern NSString * const MMEEventKeyOSVersion;
+extern NSString * const MMEEventKeyBuildType;
+extern NSString * const MMEEventKeyIsSilentCrash;
+extern NSString * const MMEEventKeyStackTrace;
+extern NSString * const MMEEventKeyStackTraceHash;
+extern NSString * const MMEEventKeyInstallationId;
+extern NSString * const MMEEventKeyThreadDetails;
+extern NSString * const MMEEventKeyAppId;
+extern NSString * const MMEEventKeyAppVersion;
+extern NSString * const MMEEventKeyAppStartDate;
+extern NSString * const MMEEventKeyCustomData;
+
+#pragma mark - MMEErrorDomain
+
+/*! @brief NSErrorDomain for MapboxMobileEvents */
+extern NSErrorDomain const MMEErrorDomain;
+
+/*! @brief MMEErrorDomain No Error Code */
+extern NSInteger const MMENoError;
+
+/*! @brief MMEErrorDomain Min Error Code */
+extern NSInteger const MMEErrorMin;
+
+/*! @brief MMEErrorDomain Error Code for exceptions */
+extern NSInteger const MMEErrorException;
+
+/*! @brief MMEErrorDomain Error Code for intilizing events */
+extern NSInteger const MMEErrorEventInit;
+
+/*! @brief key for MMEErrorDomain userInfo dictionary containing the description of the error */
+extern NSString * const MMEErrorDescriptionKey;
+
+/*! @brief key for MMEErrorEventInit userInfo dictionary containing the attributes which failed to create the event */
+extern NSString * const MMEErrorEventAttributesKey;
+
+/*! @brief key for MMEErrorDomain userInfo dictionary containing the underlying exception which triggered the error */
+extern NSString * const MMEErrorUnderlyingExceptionKey;
+
+#pragma mark - Deprecated
+
+extern NSString * const MMEEventKeyVendorID MME_DEPRECATED_MSG("Use MMEEventKeyVendorId");
```

Comment:
I’d be surprised if HeaderDoc could handle this macro. jazzy can’t display the deprecation replacement (realm/jazzy/issues/1072), but at least it can detect and style the deprecation message.